pkg/controller/logstorage/secrets/secret_controller.go

```diff
@@ -238,70 +244,71 @@ func (r *SecretSubController) Reconcile(ctx context.Context, request reconcile.R
 	// - Elasticsearch has its own CA and KeyPair, that is global.
 	// So, we create two certificate managers - one for managing per-tenant keypairs signed with the per-tenant CA,
 	// and another for cluster-scoped keypairs signed with the root operator CA.
-	var clusterCM, appCM certificatemanager.CertificateManager
+	var operatorSigner, cm certificatemanager.CertificateManager
 
-	// Cluster-scoped certificate manager, used for managing Elasticsearch secrets.
-	clusterCM, err = certificatemanager.Create(r.client, install, r.clusterDomain, common.OperatorNamespace(), certificatemanager.WithLogger(reqLogger))
+	// Create a cluster-scoped certificate manager from the tigera-operator CA, used for signing KeyPairs for use by Elasticsearch.
+	operatorSigner, err = certificatemanager.Create(r.client, install, r.clusterDomain, common.OperatorNamespace(), certificatemanager.WithLogger(reqLogger))
 	if err != nil {
 		r.status.SetDegraded(operatorv1.ResourceReadError, "Error building certificate manager", err, reqLogger)
 		return reconcile.Result{}, err
 	}
-	clusterCM.AddToStatusManager(r.status, render.ElasticsearchNamespace)
+	operatorSigner.AddToStatusManager(r.status, render.ElasticsearchNamespace)
```

Comment:
Does this apply for multi-tenant clusters ?

Reply:
I think it is essentially irrelevant for multi-tenant clusters. It's only relevant when using the CSR API, which we don't ever do for multi-tenant.